src/tests/Microsoft.VisualStudio.Composition.Tests/ComposableCatalogTests2.cs

```diff
@@ -302,6 +302,17 @@ public void GetAssemblyInputs_Empty()
             this.AssertExpectedInputAssemblies(expected, actual);
         }
 
+        [Theory]
+        [MemberData(nameof(DiscoveryEnginesTheoryData))]
+        public async Task GetAssemblyInputs_WithImports(PartDiscovery discovery)
+        {
+            var catalog = TestUtilities.EmptyCatalog.AddParts(
+                await discovery.CreatePartsAsync(
+                    typeof(ExportingType),
+                    typeof(ImportingType)));
+            catalog.GetInputAssemblies();
```

Comment:
This test isn't asserting anything - should you call this.AssertExpectedInputAssemblies? Are is this just testing to make sure there aren't any exceptions thrown when calling GetInputAssemblies?

Reply:
Right: `GetInputAssemblies()` was throwing an NRE because we had no test for importing types in this class. So I added one, and this test just includes that and calls the method to ensure it doesn't throw.